test/unit/dynamic-variables.test.js

```diff
@@ -0,0 +1,111 @@
+var _ = require('lodash'),
+    dynamicVariables = require('../../lib/superstring/dynamic-variables'),
+    expect = require('chai').expect;
+
+describe('Dynamic variable', function () {
+    it('should have all attributes', function () {
+        _.forOwn(dynamicVariables, function (dynamicVariable) {
+            expect(dynamicVariable).to.be.an('object');
+            expect(dynamicVariable.description).to.be.a('string');
+            expect(dynamicVariable.generator).to.be.a('function');
+        });
+    });
+
+    it('should not be a duplicate', function () {
```

Comment:
This test is no longer required because we moved away from the faker-map strategy and now the value is stored as an object.